src/06-sort.js

```diff
@@ -23,7 +25,16 @@ function sortByRuntimeAscending(songs) {}
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]}
  */
-function sortByArtistNameDescending(songs) {}
+function sortByArtistNameDescending(songs) {
+  return songs.sort((a,b) => {
+    switch (false) {
```

Comment:
Make sure to include the possibility that a === b just to leave no room for error